.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,88 @@
+name: Bug Report
+description: Report a problem and help us fix it.
+labels: ["kind/bug"]
+body:
+  - type: dropdown
+    id: Component
+    attributes:
+      label: Component
+      description: In which component did the issue occur?
+      options:
+        - Not sure*
+        - Operate*
+        - Optimize*
+        - Tasklist*
+        - Identity*
+        - Zeebe*
+        - C8Run*
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: steps-reproduction
+    attributes:
+      label: Steps to reproduce
+      description: Clear steps that allow us to reproduce the error.
+      value: |
+             1. do this
+             2. do that
+             3. now this happens
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen. 
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: 'The environment on which this error was observed.'
+      value: |
+             - OS: [e.g. MacOS 10.2, Windows 10]
+             - Component Version: [e.g. 2.0.0]
+             - Execution Platform: [e.g. Camunda 8 SaaS or SM]
```

Comment:
Here I'd like "Database: [ES/OS, relational]" as an option, as that should be a common environment parameter